contracts/test-alt/Marketplace.js

```diff
@@ -385,28 +395,66 @@ describe('Marketplace.sol', async function() {
 
     describe('dispute without refund (Eth)', function() {
       it('should resolve in favor of buyer (no commission)', async function() {
-        ({ listingID, offerID, balance: balanceBefore } = await helpers.disputedOffer({}));
-        ({ balance: balanceAfter } = await helpers.giveRuling({ listingID, offerID, ruling: 1 }))
+        ({
+          listingID,
+          offerID,
+          balance: balanceBefore
+        } = await helpers.disputedOffer({}))
+        ;({ balance: balanceAfter } = await helpers.giveRuling({
```

Comment:
The leading `;` looks a little odd. I assume that's the formatter's work.

Reply:
Yup... can probably make that look nicer